client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -601,6 +609,43 @@
         };
     }
 
+    PackagingEmbeddedDirective.$inject = ['packages'];
+    function PackagingEmbeddedDirective(packages) {
+        return {
+            templateUrl: 'scripts/superdesk-packaging/views/packaging.html',
+            require: '^sdAuthoringContainer',
+            scope: {
+                item: '='
+            },
+            link: function(scope, elem, attrs, authoringCtrl) {
+                scope.$watch('item', function(item) {
+                    if (item && item.lockIt){
+                        scope.lock();
+                    } else {
+                        scope.origItem = null;
+                        scope.$applyAsync(function() {
```

Comment:
with `$applyAsync` it will only do this in next `$digest` cycle, so `origItem` will be `null` in current cycle and it will destroy `sd-authoring` directive which has `ng-if` on it. so it will effectively reset that directive inside.
not sure if it's needed for packaging, it was done for authoring.